frontend/dashboard/pages/PageLayout/DashboardHeader/DashboardHeader.test.tsx

```diff
@@ -202,6 +211,78 @@ describe('DashboardHeader', () => {
     expect(mockNavigate).toHaveBeenCalledTimes(1);
   });
 
+  it('should not render the submenu when showSubMenu is false', () => {
+    (useParams as jest.Mock).mockReturnValue({
+      selectedContext: mockOrgTtd,
+      subroute: Subroute.OrgLibrary,
+    });
+    (useLocation as jest.Mock).mockReturnValue({ pathname: mockPathnameOrgLibraryTtd });
+
+    renderDashboardHeader({ showSubMenu: false, isRepoError: false });
+
+    expect(getFetchChangesButton()).not.toBeInTheDocument();
+  });
+
+  it('should not render the submenu when isRepoError is true', () => {
+    (useParams as jest.Mock).mockReturnValue({
+      selectedContext: mockOrgTtd,
+      subroute: Subroute.OrgLibrary,
+    });
+    (useLocation as jest.Mock).mockReturnValue({ pathname: mockPathnameOrgLibraryTtd });
+
+    renderDashboardHeader({ showSubMenu: true, isRepoError: true });
+
+    expect(getFetchChangesButton()).not.toBeInTheDocument();
+  });
+
+  it('should not render the submenu when the page is not orgLibrary', () => {
+    (useParams as jest.Mock).mockReturnValue({
+      selectedContext: mockOrgTtd,
+      subroute: Subroute.AppDashboard,
+    });
+    (useLocation as jest.Mock).mockReturnValue({ pathname: mockPathnameOrgLibraryTtd });
+
+    renderDashboardHeader({ showSubMenu: true, isRepoError: false });
+
+    expect(getFetchChangesButton()).not.toBeInTheDocument();
+  });
+
+  it('should not render the submenu when page is orgLibrary and feature flag is not on', () => {
```

Comment:
How can we be sure the feature flag is not toggled here, given that other tests may toggle on/off the feature flag? I suggest adding a reset/clearing of feature flags to the `afterEach`-method to be sure 😊